src/setup/Microsoft.VisualStudio.ProjectSystem.Managed.CommonFiles/Microsoft.VisualStudio.ProjectSystem.Managed.CommonFiles.csproj

```diff
@@ -15,4 +15,9 @@
   <ItemGroup>
     <SwrFile Include="*.swr" />
   </ItemGroup>
+
+  <Target Name="Build" DependsOnTargets="ResolveProjectReferences">
+    <MakeDir Directories="$(IntermediateOutputPath)" ContinueOnError="True"/>
+  </Target>
+
```

Comment:
What's the purpose behind this?

Reply:
This was in Directory.Build.targets and moved to the project, I've not changed it. I think its intentions is to ensure that its project dependencies are built and that its intermediates are created for when we generate vsman/vsix for it.